images/scripts/start-kafka.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export KAFKA_CONFIGURATION=$(cat /config/server.properties)
+${KAFKA_HOME}/generate-configuration.sh > /tmp/server.properties
```

Comment:
just curious why not to use e.g. `${KAFKA_HOME}/server.properties` - then it'll be visible in the sandbox for troubleshooting purposes

Reply:
I was having issues with UID in the start, so I just used that to go ahead. Now moved to `${KAFKA_HOME}`